packages/@aws-cdk/aws-msk-alpha/lib/cluster-version.ts

```diff
@@ -119,6 +142,16 @@ export class KafkaVersion {
     return new KafkaVersion(version);
   }
 
+  /**
+   * List of Kafka versions that support tiered storage
+   *
+   * @see https://docs.aws.amazon.com/msk/latest/developerguide/msk-tiered-storage.html#msk-tiered-storage-requirements
+   */
+  private static readonly TIERED_STORAGE_COMPATIBLE_VERSIONS = [
```

Comment:
Can we also update the [README](packages/@aws-cdk/aws-msk-alpha/lib/cluster-version.ts) here to mention that 3.6.0 also supports tiered storage.

Reply:
Done, thanks